src/data/defaults/progress/subMachineGuns.js

```diff
@@ -52,6 +52,6 @@ export default {
 		'Motley': false,
 		'Eclipse': false,
 		'Feral Beast': false,
-		'2087': false,
+		'2087 ': false,
```

Comment:
Won't this break the current stored progress for all users that have set this to true? 🤔

Reply:
Yeah it will